kube-client/src/api/portforward.rs

```diff
@@ -62,6 +61,9 @@ pub enum Error {
     /// Failed to receive a WebSocket message from the server.
     #[error("failed to receive a WebSocket message: {0}")]
     ReceiveWebSocketMessage(#[source] ws::Error),
+
+    #[error("failed to complete the background task: {0}")]
+    Spawn(#[from] tokio::task::JoinError),
```

Comment:
We've been avoiding `#[from]` because it's difficult to find where the error happens (#688).
```suggestion
    Spawn(#[source] tokio::task::JoinError),
```